src/containers/Article.js

```diff
@@ -188,7 +188,14 @@ export default class Article extends Component {
             relateds={article.relateds}
           />
         </div>
-
+        <ArticleComponents.PageNavigation
+          article={ _.get(article, [ 'relateds', 0 ])}
+          navigate="next"
+        />
+        <ArticleComponents.PageNavigation
+          article={_.get(article, [ 'relateds', 1 ])}
+          navigate="previous"
+        />
```

Comment:
Sorry, it's just a temporary solution for showing next and previous articles.
It should be discussed later after the designer and HC come into office.